README.md

```diff
@@ -281,8 +281,13 @@ In our example we put the following content in it:
 example-key: example-value
 ```
 
-NOTE: At the moment we do not support nested data structures that
-sops support. This might change in the future. See also [Different file formats](#different-file-formats)
+Nesting the key results in the creation of directories.
+These directories will be owned by root:root and have permissions 0755.
```

Comment:
So this is what are currently doing for the root secret directory, and I think we should stick
with it for nesting:

```suggestion
These directories will be owned by root:keys and have permissions 0751.
```